base_user_role_company/tests/test_use_only_enabled_roles.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2021 Open Source Integrators
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from datetime import timedelta
+
+from odoo import fields
+from odoo.tests.common import TransactionCase
+
+
+class TestUserRoleCompany(TransactionCase):
+    def setUp(self):
```

Comment:
Please refactor to use 
```
@classmethod
def setUpClass(cls)
```
so that it will only have run once to be used for both tests.